packages/css-library/src/utilities/position.scss

```diff
@@ -0,0 +1,20 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use './settings' as *;
+
+$u-position: (
+  position: (
+    base: 'vads-position',
+    modifiers: null,
+    values:
+      map-collect(
+        (
+          absolute: absolute,
+          fixed: fixed,
+          relative: relative,
+        )
+      ),
```

Comment:
May want these to be defined as a SASS variable to be consistent with others.

Reply:
These now exist in `src/tokens/position.scss`.